src/pull-request/pull-request-general-information.html

```diff
@@ -62,7 +115,8 @@
         name: String
       },
       observers: [
-        '_fireAjax(pullRequest)'
+        '_fireAjax(pullRequest)',
+        'updateStyles(pullRequest.state)'
```

Comment:
We might want to add a comment here why this is necessary, preferably with a link to the documentation section that describes this limitation.